crates/http-source/src/websocket_source.rs

```diff
@@ -128,11 +130,23 @@ async fn websocket_writer_and_stream<'a>(
 impl WebSocketSource {
     pub(crate) fn new(config: &HttpConfig) -> Result<Self> {
         let ws_config = config.websocket_config.as_ref();
+        let subscription_messages = if let Some(ws_config) = ws_config {
+            if let Some(messages) = ws_config.subscription_messages.as_ref() {
+                Some(messages.clone())
+            } else if let Some(message) = ws_config.subscription_message.as_ref() {
+                warn!("`subscription_message` is deprecated, please use `subscription_messages` instead");
+                Some(vec![message.to_owned()])
+            } else {
+                None
+            }
+        } else {
+            None
+        };
```

Comment:
I think lets accept both for now & keep the deprecation message.  I would suggest always start with an empty vec and push or append any supplied values for `subscription_message` or `subscription_messages`, with `subscription_message` at the beginning if provided.

This pr should probably also bump the connector version to 0.4.0 and refresh the README.md with that version too.  Between this and the other PR it's a nice upgrade!

Reply:
Updated to the `Vec<String>`, defaulting to `vec![]`, if `subscription_message` is provided then `vec![subscription_message]`, and if both then `vec![subscription_message] + subscription_messages`.